src/config/rcxml.c

```diff
@@ -54,6 +58,59 @@ enum font_place {
 static void load_default_key_bindings(void);
 static void load_default_mouse_bindings(void);
 
+/* Does a boolean-parse but also allows 'unset' */
+static void
+set_property(const char *str, enum property *variable)
+{
+	if (!str || !strcasecmp(str, "unset")) {
+		*variable = LAB_PROP_UNSPECIFIED;
+		return;
+	}
+	int ret = parse_bool(str, -1);
+	if (ret < 0) {
+		return;
+	}
+	*variable = ret ? LAB_PROP_TRUE : LAB_PROP_FALSE;
+}
+
+static void
+fill_window_rule(char *nodename, char *content)
+{
+	if (!strcasecmp(nodename, "windowRule.windowRules")) {
+		current_window_rule = znew(*current_window_rule);
+		wl_list_append(&rc.window_rules, &current_window_rule->link);
+		wl_list_init(&current_window_rule->actions);
+		return;
+	}
+
+	string_truncate_at_pattern(nodename, ".windowrule.windowrules");
+	if (!content) {
+		/* nop */
+	} else if (!current_window_rule) {
+		wlr_log(WLR_ERROR, "no window-rule");
+	} else if (!strcmp(nodename, "identifier")) {
+		free(current_window_rule->identifier);
+		current_window_rule->identifier = xstrdup(content);
+	} else if (!strcmp(nodename, "event")) {
+		if (!strcasecmp(content, "onMap")) {
+			current_window_rule->event = LAB_WINDOW_RULE_EVENT_ON_MAP;
+		}
+	} else if (!strcmp(nodename, "serverdecoration")) {
```

Comment:
Mixed use of `strcmp` and `strcasecmp` in this function, especially for the term `serverDecorations` I think we should use `strcasecmp` instead.

Reply:
Have changed to `strcasecmp` and `serverDecoration` because it's easier to read.

`nodename` is always lower-case so not change in functionality - just easier to read (at the expense of a few CPU cycles).